core/input.ts

```diff
@@ -37,10 +37,9 @@ import {RenderedConnection} from './rendered_connection.js';
  * @alias Blockly.Input
  */
 export class Input {
-  static Align: AnyDuringMigration;
   private sourceBlock_: Block;
   fieldRow: Field[] = [];
-  align: number;
```

Comment:
If you decided to also export these as module exports, why change the name to use the namespace identifier? I.E. why use `Input.Align` instead of `Align`?